Project-1/README.md

```diff
@@ -63,60 +68,228 @@ T	T	F	T	T
+
   ```(a)  B(x,y) that says that x is a brother of y
   (b)  A(x,y) that says that x is an aunt of y
   (c)  C(x,y) that says that x and y are cousins 
   (d)  O(x) that says that x is an only child
   (e)  T(x) that says that x has exactly two brothers 
-```
+
+```(a) ∃x∃y∃z(P(z,y)∧P(z,x)∧¬Fx) - There is some x and some y and some z such that z is the parent of y and z is the parent x and x is not female
+(b) ∃w∃x∃y∃z(P(z,y)∧P(w,z)∧P(w,x)∧Fx) - There is some w and some x and some y and some z such that z is the parent of y and w is the parent of z and w is the parent of x and x is female
+(c) ∃v∃w∃x∃y∃z(P(v,w)∧P(v,z)∧P(w,x)∧P(z,y)) - There is some v and some w and some x and some y and some z such that v is the parent of w and v is the parent of z and w is the parent of x and z is the parent of y
+(d) ∃x∃y∀z(P(y,z)→z=x) - There is some x and some y such that for all z if y is the parent of z then z and x are identical 
+(e) ∃v∃w∃x∃y∀z(P(v,x)∧P(v,w)∧¬Fw∧P(v,y)∧¬Fy∧∀z((P(v,z)∧¬Fz)→((z=x)∨(z=w)∨(z=y)))) - There is some v and some w and some x and some y such that for all z if v is the parent of x and v is the parent of w and w is not female and v is the parent of y and y is not female and for all instances of z v is a parent of z and z is not female ten z is identical with w or z is identical with w or z is identical with y
 
 4. Let V be a vocabulary of the attribute (concept) language with complements (ALC) consisting of a role name "parent_of" and a concept name "Male". Interpret parent_of as "x is a parent of y" and M as "x is male". Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   ```(a)  B that says that x is a brother of y
   (b)  A that says that x is an aunt of y
   (c)  C that says that x and y are cousins 
   (d)  O that says that x is an only child
   (e)  T that says that x has exactly two brothers 
-```
 
-5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+```(a) ΔI = x,y,z
+(a) parent_of = (z,x) (z,y)
+(a) Male = x
+(b) ΔI = w, x, y, z
+(b) praren_of = (w,x) (w,z) (z,y)
+(b) ¬Male = x
+(c) ΔI = v, w, x, y, z
+(c) parent_of = (v, w) (v,z) (w,x) (z,y)
+(d) Not possible in this language as saying x is an only child of y but would make everything else false.
+(e) Not possible in this laguage as you can say that x has two brothers but you can't say that x has exactly two brothers.  
+
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both.
+
+ TBox = {Brother ⊆ Male,		(TBox.1)
```

Comment:
It seems that "⊆" is not a symbol of description logic. Maybe you can use Latex code $\sqsubseteq$ instead by inserting \sqsubseteq into two $ symbols (no spacing).

Reply:
Thank you!